h5netcdf/core.py

```diff
@@ -474,7 +485,7 @@ def _require_child_group(self, name):
 
     def create_group(self, name):
         if name.startswith("/"):
-            return self._root.create_group(name[1:])
+            return self._root.create_group(name[1:], track_order=self._track_order)
```

Comment:
Here track_order isn't necessary, it's calling the (same) method of the root-object.

Reply:
Thanks!